node/network/availability-recovery/src/lib.rs

```diff
@@ -162,9 +168,12 @@ struct RequestChunksFromValidators {
 	/// a random shuffling of the validators which indicates the order in which we connect to the validators and
 	/// request the chunk from them.
 	shuffling: VecDeque<ValidatorIndex>,
-	received_chunks: HashMap<ValidatorIndex, ErasureChunk>,
+	/// Chunks received so far.
+	received_chunks: Option<HashMap<ValidatorIndex, ErasureChunk>>,
```

Comment:
Why is this an `Option` now? Docs would be good - or another encoding of this superficially superfluous state, maybe an enum?

Reply:
Not really needed, I'll remove it. I was using it to temporarily hold the chunks hashmap until it is sent to the recovery task for reconstruction. I'll use std::mem::take() instead.